meson.build

```diff
@@ -356,13 +362,10 @@ else
 endif
 
 if want_libfuzzer
-        fuzzing_engine = meson.get_compiler('cpp').find_library('Fuzzer', required : false)
-        if fuzzing_engine.found()
-                userspace_c_args += '-fsanitize-coverage=trace-pc-guard,trace-cmp'
-        elif cc.has_argument('-fsanitize=fuzzer-no-link')
-                userspace_c_args += '-fsanitize=fuzzer-no-link'
+        if cc.has_argument('-fsanitize=fuzzer-no-link')
+                userspace_c_args += f'-fsanitize=fuzzer-no-link@sanitizers@'
```

Comment:
I don't think it should be necessary to use `@sanitizers@`. I think `meson` should do the right thing without the `@sanitizers@` part when `b_sanitize` or `CFLAGS/LDFLAGS` is used.

Reply:
Just to make sure I dropped the `@sanitizers@` part entirely and as far as I can see it works. Could it be that you ran into https://github.com/mesonbuild/meson/issues/14533 or something like that? it broke a bunch of projects on OSS-Fuzz.

Reply:
Yeah not sure what I ran into, but it does seem to work without this, removed again.